R/adducts.R

```diff
@@ -7,10 +7,13 @@
 #'
 #' @param x `numeric` neutral mass for which the adduct m/z shall be calculated.
 #'
-#' @param adduct `character` specifying the name of the adduct;
-#'     supported values are returned by [adductNames()]
+#' @param adduct `character` specifying the name(s) of the adduct(s) for which
+#'     the m/z should be calculated; supported values are returned by
+#'     [adductNames()]
 #'
-#' @return `numeric` representing the calculated m/z value.
+#' @return numeric `matrix` with same number of rows than elements in `x` and
+#'     number of columns being equal to the length of `adduct`. Each column
+#'     thus represents the m/z of `x` for each defined `adduct`.
```

Comment:
Does the matrix have column names?

Reply:
Yes it does. The column names are the provided adduct names. I will add that to the documentation.